public/index.html

```diff
@@ -29,7 +29,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Do I have the BMLT?</title>
+    <title>Do I Have The BMLT?</title>
```

Comment:
The word "the" should not be capitalized in a title. 


http://www.superheronation.com/2011/08/16/words-that-should-not-be-capitalized-in-titles/